converter/huggingface_gptj_convert.py

```diff
@@ -168,7 +168,7 @@ def split_and_convert(args):
                                                                                  ft_model_name_pattern[i])
 
                     pool.starmap(split_and_convert_process,
-                                 [(0, saved_dir, factor, new_name, args,
+                                 [(0, saved_dir, factor, new_name,
```

Comment:
Could you append the commit description  in the commit body  to maintian the history  management?

e.g., 
```
fix: fixed the excess argument in pool.starmap() funtion

The latest commit broke the HF GPTJ converter because the pool mapper calls 
split_and_convert_process with one argument too many ("args"). 

```

Reply:
Good catch @claudiosv! `args` however is not used in this function, so ideally this PR should remove args from line 171 instead of introducing it here and not using it. Could you adjust the PR?

Reply:
@claudiosv good catch!